src/__tests__/Search.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Search from '../components/Search';
+
+describe('Search', () => {
+	const validProps = {
+		setSearchResults: () => {},
+	};
+	it('renders correctly', () => {
```

Comment:
A clear line between each test would be good here. Also, could the click event on the button be tested?